src/components/common/FullPost.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+import React from 'react';
+import {View, Text, Image, StyleSheet} from 'react-native';
+import UserPost from './UserPost';
+
+type uri = {
+  uri: string
+}
+
+type Props = {
+  uri: uri,
+  likes: string,
+  description: string,
+  userName: string
+};
+
+const FullPost = (props: Props) => {
+  const {likes, description, userName, uri} = props;
+  return (
+  <View style={styles.container}>
+    <UserPost uri={uri} userName={userName}>
+      <View style={styles.likes}>
+        <Image style={styles.img} source={require('../../assets/img/likes.png')}/>
```

Comment:
Move images import before the component:
`const imageLike = require('../../assets/img/likes.png');`
then
`<Image style={styles.img} source={imageLike}/>`